draft-irtf-cfrg-aegis-aead.md

```diff
@@ -967,11 +967,11 @@ The `Dec` function decrypts a 128-bit input block `ci` using the state `{S0, ...
 
 Inputs:
 
-- `ci`: the encrypted input block.
+- `ci`: the 128-bit encrypted input block.
```

Comment:
`input` can be removed to match the [AEGIS-128L section](https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-aegis-aead-14#section-3.6).

Reply:
In `The Update Function` for [AEGIS-256](https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-aegis-aead-14#section-4.4), the following could also be changed to match the [corresponding AEGIS-128L section](https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-aegis-aead-14#section-3.4):

`msg: the block to be absorbed.` => `msg: the 128-bit block to be absorbed.`

Reply:
Good catch. We should be consistent. Changed, thanks!